specifyweb/frontend/js_src/lib/localization/common.ts

```diff
@@ -616,4 +616,10 @@ export const commonText = createDictionary({
     'ru-ru': 'Времени осталось',
     'uk-ua': 'Час, що залишився',
   },
+  change: {
+    'en-us': 'Change',
+  },
+  pickImage: {
```

Comment:
can you move `pickImage` into `preferencesText` localization dictionary. it doesn't really belong in `commonText`

`change` is fine in here as it's something that could theoretically be used anywhere